.github/workflows/build.yml

```diff
@@ -253,17 +253,20 @@ jobs:
       os-matrix: '["ghcr.io/cirruslabs/macos-runner:sonoma", "macos-14", "macos-13"]'
 
   build_ubuntu:
-    name: 'Ubuntu'
+    name: >-
+      Ubuntu
+      ${{ fromJSON(matrix.free-threading) && '(free-threading)' || '' }}
     needs: check_source
     if: needs.check_source.outputs.run_tests == 'true'
+    strategy:
+      matrix:
+        free-threading:
```

Comment:
There's no free-threading in 3.12. 

The Windows 3.12 backport included free-threading things in reusable-windows.yml but build.yml didn't mention it.

Reply:
... because the https://github.com/python/cpython/pull/121776 backport hasn't been merged yet :) 

But that comments out the `free-threading: true` job. Do the same here?

Reply:
> But that comments out the `free-threading: true` job. Do the same here?

Yes! I forgot about that one, only fixed the merge conflicts. I left it commented out to facilitate any future backports, reducing possible conflicts.

Reply:
I was wondering why that doesn't fail the CI and saw this:
```console
configure: WARNING: unrecognized options: --disable-gil
```

@hugovk any reason invalid flags don't crash the build process?

Reply:
Commented out that matrix factor. Marking this thread as resolved.

Reply:
> I was wondering why that doesn't fail the CI and saw this:
> 
> ```
> configure: WARNING: unrecognized options: --disable-gil
> ```
> 
> @hugovk any reason invalid flags don't crash the build process?

Let's ask @erlend-aasland